src/pybind/mgr/pg_autoscaler/test_pg_autoscaler.py

```diff
@@ -0,0 +1,79 @@
+#python unit test
+import unittest
+import os
+import sys
+from tests import mock
+
+import pytest
+import json
+os.environ['UNITTEST'] = "1"
+sys.path.insert(0, "../../pybind/mgr")
```

Comment:
Have a look at 

https://github.com/ceph/ceph/blob/2cd46a141c46d95b378332c3e89da9b0ee77a67b/src/pybind/mgr/cephadm/__init__.py

for an alternative to this